ricoconverter/ricoconverter-convert/src/test/resources/ead2rico/_15b_level_hasOrHadPart_hasOrHadConstituent/expected.xml

```diff
@@ -128,5 +129,31 @@
           </rico:RecordResource>
         </rico:hasOrHadPart>
     </rico:RecordResource>
-      
+
+    <!-- cas 3.4 : c RecordPart -->
+    <rico:RecordResource rdf:about="recordResource/058220-a3.4">
+        <rico:isOrWasIncludedIn rdf:resource="recordResource/top-058220"/> 
+        <rico:hasInstantiation>
+          <rico:Instantiation rdf:about="instantiation/058220-a3.4-i1">
+            <rico:isInstantiationOf rdf:resource="recordResource/058220-a3.4"/>
+            <rdfs:seeAlso rdf:resource="https://www.siv.archives-nationales.culture.gouv.fr/siv/UD/FRAN_IR_058220/a3.4"/>
+          </rico:Instantiation>
+        </rico:hasInstantiation>
+        <rico:hasOrHadPart>
+          <rico:RecordResource rdf:about="recordResource/058220-a3.4-child">
+            <rico:isOrWasPartOf rdf:resource="recordResource/058220-a3.4"/>
+            <!-- This is a RecordPart, hence using isOrWasPartOf -->
```

Comment:
Yes, unless we know that the current Record Resource is a Record; if it is a record, we can have rico:hasOrHadConstituent. Maybe we could add another case in the input and expected files to specify this.

Reply:
See my comment to the input.xml file